### ISSUE TITLE
table.id()

### ISSUE THREAD

**Issue by thosakwe** [thosakwe/furlong#1]
There should be an easy 'table.id()' method so that models can have ID fields across different migrators, so that code generation works without hassle.

**Comment by thosakwe** [comment]
For example:

```dart
class MySqlTableSchema extends TableSchema {
  FieldGenerator id([String idField]) => primaryKey = integer(idField ?? "id", autoIncrement: true);
}
```

**Comment by thosakwe** [comment]
Resolved with commit 3711f6b285907e3fd6fd31aec23064e8d30ecc85
